packages/extension-base/src/background/handlers/Extension.ts

```diff
@@ -239,7 +239,6 @@ export default class Extension {
     return { list: this.#state.authUrls };
   }
 
-  // FIXME This looks very much like what we have in accounts
```

Comment:
fixme suggests refactor - I'm pretty content with how it looks currently, removing comment